README.md

```diff
@@ -40,25 +40,22 @@ See the [Installation](https://docs.all-hands.dev/modules/usage/installation) gu
 system requirements and more information.
 
 ```bash
-export WORKSPACE_BASE=$(pwd)/workspace
```

Comment:
We are removing this to encourage the github or upload zip workflows? But setting your workspace is also possible?

Reply:
Or is the place to not support it at all going forward?

Reply:
Correct--we're no longer encouraging it (since it causes so many problems with file permissions and complicates the install), but we will continue to support it